moto/apigateway/models.py

```diff
@@ -126,14 +126,18 @@ def __init__(
         uri,
         http_method,
         request_templates=None,
+        passthrough_behavior="WHEN_NO_MATCH",
+        cache_key_parameters=None,
```

Comment:
This value is never read, so this always defaults to the empty list. Was that on purpose?

Looks like the test also fails because of this field.

Reply:
The cache_key_parameters defaults to empty list, at least the boto response documentation says so and by observation. The tests are failing because I missed to include the field in the assertion.